### ISSUE TITLE
Build release 23.11.0 manually

### ISSUE THREAD

**Issue by Maleware** [stackabletech/issues#483]
Checking out the actual status for arm architecture, we would like to build the release 23.11.0 from scratch and manually and provide it in nexus.

This is to enable the build of the long living cluster in parallel to other tasks for arm.

**Comment by Maleware** [comment]
Findings:

We have a new Issue with HBase:
`[WARNING] Rule 0: org.apache.maven.plugins.enforcer.EvaluateBeanshell failed with message:
287.4 License errors detected, for more detail find ERROR in
287.4                     /stackable/hbase-2.4.12-src/hbase-assembly/target/maven-shared-archive-resources/META-INF/LICENSE
287.4 [INFO] ------------------------------------------------------------------------
287.4 [INFO] Reactor Summary:
287.4 [INFO] 
287.4 [INFO] Apache HBase 2.4.12 ................................ SUCCESS [ 30.439 s]
287.4 [INFO] Apache HBase - Checkstyle .......................... SUCCESS [  6.545 s]
287.4 [INFO] Apache HBase - Annotations ......................... SUCCESS [  1.170 s]
287.4 [INFO] Apache HBase - Build Configuration ................. SUCCESS [  0.239 s]
287.4 [INFO] Apache HBase - Logging ............................. SUCCESS [  0.981 s]
287.4 [INFO] Apache HBase - Shaded Protocol ..................... SUCCESS [ 27.340 s]
287.4 [INFO] Apache HBase - Common .............................. SUCCESS [ 25.985 s]
287.4 [INFO] Apache HBase - Metrics API ......................... SUCCESS [  1.742 s]
287.4 [INFO] Apache HBase - Hadoop Compatibility ................ SUCCESS [  1.483 s]
287.4 [INFO] Apache HBase - Metrics Implementation .............. SUCCESS [  0.980 s]
287.4 [INFO] Apache HBase - Hadoop Two Compatibility ............ SUCCESS [  7.595 s]
287.4 [INFO] Apache HBase - Protocol ............................ SUCCESS [  7.831 s]
287.4 [INFO] Apache HBase - Client .............................. SUCCESS [  5.461 s]
287.4 [INFO] Apache HBase - Zookeeper ........................... SUCCESS [  1.311 s]
287.4 [INFO] Apache HBase - Replication ......................... SUCCESS [  1.252 s]
287.4 [INFO] Apache HBase - Resource Bundle ..................... SUCCESS [  0.088 s]
287.4 [INFO] Apache HBase - HTTP ................................ SUCCESS [ 12.830 s]
287.4 [INFO] Apache HBase - Asynchronous FileSystem ............. SUCCESS [  2.921 s]
287.4 [INFO] Apache HBase - Procedure ........................... SUCCESS [  1.517 s]
287.4 [INFO] Apache HBase - Server .............................. SUCCESS [ 23.963 s]
287.4 [INFO] Apache HBase - MapReduce ........................... SUCCESS [  4.483 s]
287.4 [INFO] Apache HBase - Testing Util ........................ SUCCESS [  2.476 s]
287.4 [INFO] Apache HBase - Thrift .............................. SUCCESS [  7.079 s]
287.4 [INFO] Apache HBase - RSGroup ............................. SUCCESS [  5.310 s]
287.4 [INFO] Apache HBase - Shell ............................... SUCCESS [  5.455 s]
287.4 [INFO] Apache HBase - Coprocessor Endpoint ................ SUCCESS [  5.845 s]
287.4 [INFO] Apache HBase - Integration Tests ................... SUCCESS [ 12.838 s]
287.4 [INFO] Apache HBase - Rest ................................ SUCCESS [  4.481 s]
287.4 [INFO] Apache HBase - Examples ............................ SUCCESS [  4.890 s]
287.4 [INFO] Apache HBase - Shaded .............................. SUCCESS [  0.219 s]
287.4 [INFO] Apache HBase - Shaded - Client (with Hadoop bundled) SUCCESS [ 19.400 s]
287.4 [INFO] Apache HBase - Shaded - Client ..................... SUCCESS [ 10.263 s]
287.4 [INFO] Apache HBase - Shaded - MapReduce .................. SUCCESS [ 17.130 s]
287.4 [INFO] Apache HBase - External Block Cache ................ SUCCESS [  1.816 s]
287.4 [INFO] Apache HBase - HBTop ............................... SUCCESS [  1.254 s]
287.4 [INFO] Apache HBase - Assembly ............................ FAILURE [  6.318 s]
287.4 [INFO] Apache HBase - Shaded - Testing Util ............... SKIPPED
287.4 [INFO] Apache HBase - Shaded - Testing Util Tester ........ SKIPPED
287.4 [INFO] Apache HBase Shaded Packaging Invariants ........... SKIPPED
287.4 [INFO] Apache HBase Shaded Packaging Invariants (with Hadoop bundled) SKIPPED
287.4 [INFO] Apache HBase - Archetypes .......................... SKIPPED
287.4 [INFO] Apache HBase - Exemplar for hbase-client archetype . SKIPPED
287.4 [INFO] Apache HBase - Exemplar for hbase-shaded-client archetype SKIPPED
287.4 [INFO] Apache HBase - Archetype builder 2.4.12 ............ SKIPPED
287.4 [INFO] ------------------------------------------------------------------------
287.4 [INFO] BUILD FAILURE
 `

**Comment by Maleware** [comment]
Commons operator pulls libraries with wrong architecture
` Downloaded windows_x86_64_gnullvm v0.48.0
  Downloaded object v0.31.1
  Downloaded regex v1.9.3
  Downloaded regex-automata v0.3.6
  Downloaded windows_i686_msvc v0.48.0
  Downloaded windows_aarch64_msvc v0.48.0
  Downloaded libc v0.2.150
  Downloaded linux-raw-sys v0.4.5
  Downloaded windows_x86_64_msvc v0.48.0
  Downloaded windows_x86_64_gnu v0.48.0
  Downloaded windows_i686_gnu v0.48.0`

**Comment by Maleware** [comment]
Listener:
`error: failed to run custom build command for `csi-grpc v23.11.0 (/src/rust/csi-grpc)`
141.4 
141.4 Caused by:
141.4   process didn't exit successfully: `/src/target/release/build/csi-grpc-b35efe538fb734e1/build-script-build` (exit status: 101)
141.4   --- stdout
141.4   cargo:rerun-if-changed=csi.proto
141.4   cargo:rerun-if-changed=vendor/csi
141.4 
141.4   --- stderr
141.4   thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Custom { kind: Other, error: "protoc failed: Could not make proto path relative: csi.proto: No such file or directory\n" }', rust/csi-grpc/build.rs:10:10
141.4   note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
141.4 warning: build failed, waiting for other jobs to finish...
------
Dockerfile:1
--------------------
   1 | >>> # =============
   2 |     # This file is automatically generated from the templates in stackabletech/operator-templating
   3 |     # DON'T MANUALLY EDIT THIS FILE
--------------------
ERROR: failed to solve: process "/bin/bash -o pipefail -c . \"$HOME/.cargo/env\" && cargo auditable build --release --workspace && cargo cyclonedx --output-pattern package --all --output-cdx" did not complete successfully: exit code: 101
make: *** [docker-build] Error 1
`

**Comment by Maleware** [comment]
Images should be available [here](https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable-experimental)

**Comment by lfrancke** [comment]
https://issues.apache.org/jira/browse/HBASE-27065
https://issues.apache.org/jira/browse/HBASE-27103
those are the two related HBase issues
